### ISSUE TITLE
Packages not present after stop/starting Tensorflow Image Notebook

### ISSUE THREAD

**Issue by maganaluis** [awslabs/kubeflow-manifests#760]
**Describe the bug**
Whenever we use a Tensorflow based image like the one below, it does not store the packages to the local directory, so after stopping the Notebook the packages installed are not longer there. This causes issues whenver these notebooks need to be stopped or migrated for whatever reason.

**Steps To Reproduce**
1. Start a notebook using the image `public.ecr.aws/kubeflow-on-aws/notebook-servers/jupyter-tensorflow:2.12.0-cpu-py310-ubuntu20.04-ec2-v1.0` 
2. Install some package, say `numpy==1.25.0` and perform `pip freeze | grep numpy`, note the version.
3. Stop and Start the Notebook
4. Perform `pip freeze | grep numpy`.

**Expected behavior**
I expect to see the packages installed on the previous session in the notebook after stoping and starting the notebook.

**Environment** 
* Kubernetes version: 1.23
* Using EKS (yes/no), if so version? yes
* Kubeflow version: 1.6.1
* AWS build number: See steps to reproduce 
* AWS service targeted (S3, RDS, etc.): ECR? N/A?

**Screenshots**
If applicable, add screenshots to help explain your problem. N/A

**Additional context**

**Comment by maganaluis** [comment]
I looked into this myself

- On the Pytorch image the user does not have access to install in the root directory only to `/home/jovyan` so pip by default installs them on the home directory and therefore are presisted.
- On the Tensorflow image the user has access to install in the root directory and pip installs them there but gives you a warning to install them using `--user` or something like that. If you install them using this flag, it will install them to `/home/jovyan` and persist them for your next session.

TL;DR - If you want your packages to persist on the Tensorflow Image use `pip install <package> --user`
